exporter/metric/README.md

```diff
@@ -13,26 +13,35 @@ Google Cloud Monitoring is a managed service provided by Google Cloud Platform.
 
 ## Usage
 
-Add `github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric` to the import list and set up `go.mod` file accordingly.
-
-```go
-import mexpoter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric"
-```
-
 After you import the metric exporter package, then register the exporter to the application, and start sending metrics. If you are running in a GCP environment, the exporter will automatically authenticate using the environment's service account. If not, you will need to follow the instructions in [Authentication](#Authentication).
 
 ```go
 
 import (
-    ... (omitted)
     "go.opentelemetry.io/otel/api/metric"
+    "go.opentelemetry.io/otel/label"
+    "go.opentelemetry.io/otel/sdk/metric/controller/push"
+    "go.opentelemetry.io/otel/sdk/resource"
+
+    mexporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric"
 )
 
 // Initialize exporter option.
-opts := []mexporter.Option{}
+projectID := os.Getenv("GOOGLE_CLOUD_PROJECT")
```

Comment:
As I commented in https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/issues/95#issuecomment-690782197, this shouldn't be necessary when the application is running in GCP environment.

So I would prefer to omit this in the initial example, and then add a short paragraph describing how to set / override project ID if needed.

Reply:
OK. Will revise. Would you like me to make the change to the Trace `README` example too to keep them consistent? if so, I'll submit the Trace change as a separate PR.

Reply:
Fixed: 2edc48082e9cc22b0d0c2614a612ba8fa2f6f76a